### PR TITLE
JBPM-5914: Add tests checking correct status codes

### PR DESCRIPTION
**Pull request by domhanak**
Adds test cases that check if there is a correct error information in a message
displayed to user, when specifying bad/non-existent containerId, taskId and time formats.
Replaces usages of org.junit.Assert with AssertJ statements.

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java [hunk 1]
```diff
@@ -66,71 +67,216 @@ public void resetUser() throws Exception {
...
+        Map<String, Object> parameters = new HashMap<>();
+
+        Long processInstanceId = null;
+        try {
+            processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_EVALUATION, parameters);
+            List<TaskSummary> tasks = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10);
+            TaskSummary task = tasks.get(0);
+            OrgEntities reassign = OrgEntities.builder().users(Arrays.asList(USER_JOHN)).build();
+            assertExceptionContainsCorrectMessage(() -> userTaskAdminClient.reassignWhenNotCompleted(CONTAINER_ID, task.getId(), "2sssss", reassign)
+                                                , "Error code: 500");
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java]: I think that Kie server should return HTTP status 4xx in this case (possibly 400) as request parameter cannot be parsed.

**Comment by domhanak** [comment]
Bug report related to failed tests - [JBPM-5914](https://issues.jboss.org/browse/JBPM-5914)

**Comment by domhanak** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
@markcoble since JBPM-5914 is on you I assign this PR for you too. Please include that in your work

**Comment by mswiderski** [comment]
closing this one as tests are included in https://github.com/kiegroup/droolsjbpm-integration/pull/931

**Pull request closed without merging**